console/console-init/ui/src/components/TypeAheadSelect/TypeAhead.tsx

```diff
@@ -0,0 +1,65 @@
+import React, { useState } from "react";
```

Comment:
why folder name and file name is different, is there any reason. can we have the same name if possible.

Reply:
I will rename the component once TypeAHeadSelect.tsx file is no longer in use in other components